packages/datadog-plugin-amqp10/src/index.js

```diff
@@ -78,22 +78,19 @@ function startReceiveSpan (tracer, config, link) {
   return span
 }
 
-function addTags (tracer, config, span, link) {
-  const address = link.session.connection.address
+function addTags (tracer, config, span, link = {}) {
+  const address = getHost(link)
```

Comment:
```suggestion
  const address = getAddress(link)
```

?

Or maybe should adjust function or variable names?

Reply:
(or add a comment)

Reply:
Renamed the functions to make them clearer.